### PR TITLE
feat: Add script to read collections from folder

### PR DESCRIPTION
**Pull request by ankitKumar-extendaRetail**

### CODE REVIEW

File: composite-actions/lossprevention/acceptance/action.yml [hunk 1]
```diff
@@ -46,10 +46,22 @@ runs:
...
       run: |
           npm install newman --no-save --silent
-          npx newman run ${{ inputs.collection }} \
-            --environment ${{ inputs.environment }} \
-            --env-var iamToken=$IDENTITY_TOKEN
+          IFS=',' read -ra COLLECTIONS_ARRAY <<< "${{ steps.list_collections.outputs.collections }}"
+          for collection in "${COLLECTIONS_ARRAY[@]}"; do
+            echo "Running tests for collection: $collection"
+            npx newman run "$collection" \
+              --environment tests/integration_test.staging.postman_environment.json \
```
Comment (pallavibose) [inline composite-actions/lossprevention/acceptance/action.yml:65]: Do we really need step to list entire collection. Can we reduce the number of unnecessary echo statements?

**Review by pallavibose — CHANGES_REQUESTED** [review]

File: composite-actions/lossprevention/acceptance/action.yml (hunk 1 above)
Comment (pallavibose) [reply composite-actions/lossprevention/acceptance/action.yml:65]: Also may be this is not running the collections in parallel. Is there way to make it run in parallel?

**Review by TwanVB — APPROVED** [review]

File: composite-actions/lossprevention/acceptance/action.yml (hunk 1 above)
Comment (ankitKumar-extendaRetail) [reply composite-actions/lossprevention/acceptance/action.yml]: Steps and echo statements reduced

Comment (ankitKumar-extendaRetail) [reply composite-actions/lossprevention/acceptance/action.yml]: Discussed on the parallel approach

**Review by pallavibose — APPROVED** [review]

**Pull request merged**